src/deprecates.jl

```diff
@@ -54,6 +54,7 @@ end
 
 # Deprecate `MatrixReshaped`
 const MatrixReshaped{S<:ValueSupport,D<:MultivariateDistribution{S}} = ReshapedDistribution{2,S,D}
+Base.deprecate(@__MODULE__, :MatrixReshaped)
```

Comment:
I guess we can't just use `Base.@deprecate_binding` since it's not a simple renaming?

Reply:
Yes.